qa/resources/template.conf

```diff
@@ -91,4 +91,18 @@ sparkz {
     enabled = %(REMOTE_KEY_MANAGER_ENABLED)s
     address = "%(REMOTE_SERVER_ADDRESS)s"
   }
+
+ accountMempool {
```

Comment:
move `allowUnprotectedTxs` from `mempool` to `accountMempool` pacakge.
Fix py tests accordingly.